Fika.Core/Coop/GameMode/CoopGame.cs

```diff
@@ -293,6 +416,54 @@ private async Task<LocalPlayer> CreatePhysicalBot(Profile profile, Vector3 posit
             return localPlayer;
         }
 
+        private bool TryDespawnFurthest(Profile profile, Vector3 position, CoopHandler coopHandler)
+        {
+            String botkey = GetFurthestBot(Bots, coopHandler, out float furthestDistance);
+
+            if (botkey == string.Empty)
+            {
+                return false;
+            }
+
+            if (furthestDistance > GetDistanceFromPlayers(position, GetPlayers(coopHandler)))
+            {
+#if DEBUG
+                Logger.LogWarning($"We're not despawning anything. The furthest bot is closer than the one we wanted to spawn.");
+#endif
+                return false;
+            }
+
+            //Dont despawn inside of dynamic AI range
+            if (furthestDistance < FikaPlugin.DynamicAIRange.Value * FikaPlugin.DynamicAIRange.Value) //Square it because we use sqrMagnitude for distance calculation
+            {
+#if DEBUG
+                Logger.LogWarning($"We're not despawning anything. Furthest despawnable bot is inside DynamicAI range.");
+#endif
+                return false;
+            }
+
+            var bot = Bots[botkey];
```

Comment:
I prefer explicit variables, but you can leave it as this if you wish

Reply:
I do too. This one must've sneaked past me when I changed a lot of vars to explicits.